File: mouni/python/basics/4.3.py
a=[5,1,3,8,4]
n=5
temp=0
for i in range(0,n):
	for j in range(i+1,n):
		if(a[i]>a[j]):
			temp=a[i]
			a[i]=a[j]
			a[j]=temp
for i in range(0,n):
	print a[i],

File: ganga/basics/4.3.c
#include<stdio.h>
void main()
{
	int i,j,temp=0,a[5]={9,8,5,4,4};
	for(i=0;i<5;i++)
	{
		for(j=i+1;j<5;j++)
		{  
			if(a[i]>a[j])
			{
				temp=a[i];
				a[i]=a[j];
				a[j]=temp;
			}
		}
		printf("%d",a[i]);
	}
}
